src/rules/function-url-quotes/index.js

```diff
@@ -10,18 +10,27 @@ import {
 export const ruleName = "function-url-quotes"
 
 export const messages = ruleMessages(ruleName, {
-  expected: (f = "url") => `Expected quotes around ${f} argument`,
-  rejected: (f = "url") => `Unexpected quotes around ${f} argument`,
+  expected: () => "Expected quotes",
+  rejected: () => "Unexpected quotes",
 })
 
-export default function (expectation) {
+const defaultOptions = {
```

Comment:
Can you use the `optionsMatches` util please, [like](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/index.js#L5) [so](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/index.js#L63-L67). Rather than specifying `defaultOptions`. We endeavor to keep the structure of our rules consistent for ease the maintenance burden.